templates/default/datadog.yaml.erb

```diff
@@ -82,6 +93,20 @@ agent_config = @extra_config.merge({
 
   # log agent options
   log_enabled: node['datadog']['enable_logs_agent']
+
+  # process agent options
+  process_dd_url: node['datadog']['process_agent']['url'],
```

Comment:
Note: raclette has put there endpoint within the `apm_config` section. Could we try to be consistent here? Would `dd_url` under `process_config` be better? I know this would imply changes to the process-agent, hopefully it's not too late.

Reply:
This is a good idea. I have a PR on the process-agent side in https://github.com/DataDog/datadog-process-agent/pull/109 that will hopefully make it into the RC1